src/etc/htmldocck.py

```diff
@@ -233,6 +233,16 @@ def get_commands(template):
 
             negated = (m.group('negated') == '!')
             cmd = m.group('cmd')
+            if m.group('invalid') == '!':
+                print_err(
+                    lineno,
+                    line,
+                    'Invalid command: `!@{0}{1}`, (try with `@!{1}`)'.format(
```

Comment:
Maybe say

```suggestion
                    'Invalid command: `!@{0}{1}`, (help: try with `@!{1}`)'.format(
```

to be consistent with rustc suggestion? Not a big deal though.